text/0000-unify-katello-ui.md

```diff
@@ -0,0 +1,78 @@
+- Feature Name: Unify Katello UI
+- Start Date: 2016-08-26
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+The goal of this feature is to combine [Katello](github.com/katello/katello) and [Bastion](github.com/katello/bastion)
+into a single UI project.
+
+# Motivation
+[motivation]: #motivation
+
+Currently when working on the UI in Katello a developer has to oftentimes make changes in both the Katello and Bastion
+projects in order to complete a feature.  Once Katello begins to make more use of the components from
+[angular-patternfly](https://github.com/patternfly/angular-patternfly) this problem will only become worse as there will
+be three potential UI repositories to alter in order to complete new features.
+
+The expected outcome of this feature is to reduce the complexity of working on UI code in Katello by combining projects.
+
+# Detailed design
+[design]: #detailed-design
+
+## Option 1:  Move Bastion code back into Katello
+
+Eliminate Bastion entirely and move everything in Bastion back into Katello.
```

Comment:
Would you see this as merging what is currently Bastion into `bastion_katello` or plopping it down into the `engines/` directory and keeping the code as is?